modules/validations/incompatible_source.js

```diff
@@ -18,6 +18,10 @@ export function validationIncompatibleSource() {
       id: 'google',
       regex: /google/i,
       exceptRegex: /((books|drive)\.google|google\s?(books|drive|plus))|(esri\/Google_Africa_Buildings)/i
+    },
+    {
+      id: 'mapy.cz',
+      regex: /mapy.cz/i
```

Comment:
I think this `.` should be escaped like this `\.` – see above.

Otherwise it would match any character.

Reply:
Thanks! Fixed!